scripts/zones/Pashhow_Marshlands_[S]/mobs/Virulent_Peiste.lua

```diff
@@ -0,0 +1,14 @@
+-----------------------------------
+-- Area: Pashhow Marshlands [S] (90)
+--  MOB: Virulent Peiste
+-- Note: PH for Sugaar
+-----------------------------------
+require("scripts/zones/Pashhow_Marshlands_[S]/MobIDs.lua");
+require("scripts/globals/mobs");
+
+function onMobDeath(mob, player, isKiller)
+end;
+
+function onMobDespawn(mob)
+    phOnDespawn(mob,Sugaar_PH,15,28800); -- 15% spawn chance and 8 hour respawn are guesses as no information can be found online regarding it

```

Comment:
15% seems kinda high based on how long it took me to pop it on retail (during the course of 2 days)
and remember, with all those PH's in the MobID's that's acutally more than 15% cause its 15% for every PH when on retail you only have 1 PH (which I also found the correct one in the vid)
https://youtu.be/1zB-nDvt0tA